packages/teams-js/test/public/media.spec.ts

```diff
@@ -508,67 +508,56 @@ describe('media', () => {
         }
       });
 
-      it('videoController notifyEventToHost should fail in default version of platform', () => {
-        return mobilePlatformMock.initializeWithContext(FrameContexts.content, HostClientType.android).then(() => {
-          mobilePlatformMock.setClientSupportedSDKVersion(originalDefaultPlatformVersion);
-          let error;
-          new media.VideoController().stop((e: SdkError) => {
-            error = e;
-          });
-          expect(error).not.toBeNull();
-          expect(error.errorCode).toBe(ErrorCode.OLD_PLATFORM);
-        });
-      });
-
-      it('videoController notifyEventToHost is handled successfully', () => {
-        return mobilePlatformMock.initializeWithContext(FrameContexts.content, HostClientType.android).then(() => {
-          mobilePlatformMock.setClientSupportedSDKVersion(nonFullScreenVideoModeAPISupportVersion);
-          let mediaError: SdkError;
-          new media.VideoController().stop((e: SdkError) => {
-            mediaError = e;
-          });
-
-          const message = mobilePlatformMock.findMessageByFunc('media.controller');
-          expect(message).not.toBeNull();
-          expect(message.args.length).toBe(1);
-
-          const callbackId = message.id;
-          mobilePlatformMock.respondToMessage({
-            data: {
-              id: callbackId,
-              args: [undefined],
-            },
-          } as DOMMessageEvent);
+      it('videoController notifyEventToHost should fail in default version of platform', async () => {
```

Comment:
>it('videoController notifyEventToHost should fail in default version of platform', async () => { [](http://example.com/codeflow?start=6&length=96)

I don't see any tests where you are testing the new stop() signature with no callback